import os
os.chdir('/script/script_LightningMalware/src_analyze') 
#récupère les fichiers de résultats:
if __name__ == "__main__":
    
    with open("/script/script_LightningMalware/result/result.txt","w") as file_result:
        
        try:
            with open("/script/script_LightningMalware/result/scan_result_clam.txt", "r") as f:
                
                # Afficher les résultats
                file_result.write("Résultats de l'analyse ClamAV:\n")
                file_result.write(f.read())
                file_result.write("\n")
            with open("/script/script_LightningMalware/result/scan_result_python.txt","r") as f:
                    file_result.write("\nRésultats de l'analyse script Python:")
                    file_result.write(f.read())
                    file_result.write("\n")
        except Exception as e:
            file_result.write(f"Erreur lors de l'ouverture des fichiers de résultats : {e}")